src/utils/isStandardSyntaxMediaFeature.js

```diff
@@ -16,5 +16,8 @@ export default function (mediaFeature) {
   // SCSS or Less interpolation
   if (hasInterpolation(mediaFeature)) { return false }
 
+  // SCSS variables
+  if (/\$.+?/.test(mediaFeature)) { return false }
```

Comment:
> why we support postcss-simple-vars

Because I've used it and it takes almost no effort to support it. I don't think we should drop this just because we can. It's easy to keep.